src/components/FeatureViews/Onboarding/BusinessInfo.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import TextField from '@material-ui/core/TextField';
+import Container from "@material-ui/core/Container";
+
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+import {makeStyles} from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+    Text: {
+        marginTop: '65px'
+    },
+    Subtitle: {
+        fontFamily: 'Arial, Helvetica Neue, Helvetica, sans-serif',
+        textAlign: "center",
+        marginTop: '50px',
+
+    }
+}));
+
+
+const BusinessInfo = () => {
+    const classes = useStyles();
+
+
+    return (
+        <Container>
+            <Box className={classes.Text}>
+                <Typography
+                    variant="h3">Business Information</Typography>
+                <Box fontSize="h5.fontSize" className={classes.Subtitle}>
+                    <Typography
```

Comment:
You could put this on a single line since there is just one attribute `<Typography variant="p>"`